### PR TITLE
Improve exit messages for redccd

### PR DESCRIPTION
**Pull request by simontorres**
Related to #66

### CODE REVIEW

File: pipeline/core/core.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 from astropy.modeling import (models, fitting, Model)
 from scipy import signal
 from ..info import __version__
-
+# __version__ = __import__('pipeline').__version__
```
Comment (b1quint) [inline pipeline/core/core.py]: Is this a comment line? If it is, I recommend removing it. I believe we already have too many commented lines in the code and that we should avoid adding more.

File: pipeline/images/goodman_ccd.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from .night_organizer import NightOrganizer
 from .image_processor import ImageProcessor
 from ..info import __version__
+# __version__ = __import__('pipeline').__version__
```
Comment (b1quint) [inline pipeline/images/goodman_ccd.py]: Same here.

**Comment by simontorres** [comment]
You are right, this should be taken care by #142

**Comment by simontorres** [comment]
The test are failing on version checker due to limit of request. Merging this PR since there is no conflict.

**Pull request merged**